Makefile

```diff
@@ -125,7 +125,7 @@ endif
 webapp/node_modules: $(wildcard webapp/package.json)
 ifneq ($(HAS_WEBAPP),)
 	cd webapp && node skip_integrity_check.js
-	cd webapp && $(NPM) install
+	cd webapp && $(NPM) ci
```

Comment:
Is there any side effect of running `npm ci` locally? For example I am worried it could force a re-install of all modules every time. 

I guess the better question is: do we actually call this Makefile target from CI? If not, we may leave it unchanged or devise some additional check based on env variables.

Reply:
We do run this from the CI, but I initially thought there shouldn't be a side effect. But looking into the docs, it looks like in dev, new dependencies might not get persisted to the `package-lock.json`, so I split up the webapp build step into a `webapp-ci` version. That's cleaner too, because we are already doing the `npm ci` step in the `npm-dependencies` circleci job.  A little confusing.  Thanks for the question!